config/initializers/stripe.rb

```diff
@@ -1 +1,16 @@
-Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
\ No newline at end of file
+Stripe.api_key = ENV["STRIPE_SECRET_KEY"]
+
+StripeEvent.configure do |events|
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.